idpi/share/field_mappings.yml

```diff
@@ -0,0 +1,133 @@
+FI:
+  ifs:
+    name: z
+    paramId: 129
+  cosmo:
+    paramId: 500006
+FIS:
+  # ifs does not define a field for FIS (surface of geopotential), there is only z
+  ifs:
+    name: z
+    paramId: 129
+  cosmo:
+    paramId: 500004
+FR_LAND:
+  ifs:
+    name: lsm
+    paramId: 172
+  cosmo:
+    paramId: 500054
+SDOR:
+  ifs:
+    name: sdor
+    paramId: 160
+ETADOT:
+  ifs:
+    name: etadot
+    paramId: 77
+T:
+  ifs:
+    name: t
+    paramId: 130
+  cosmo:
+    paramId: 500014
+QV:
+  ifs:
+    name: q
+    paramId: 133
+  cosmo:
+    paramId: 500035
+U:
+  ifs:
+    name: u
+    paramId: 131
+  cosmo:
+    paramId: 500028
+V:
+  ifs:
+    name: v
+    paramId: 132
+  cosmo:
+    paramId: 500030
+PS:
+  ifs:
+    name: sp
+    paramId: 134
+  cosmo:
+    paramId: 500000
+U_10M:
+  ifs:
+    name: 10u
+    paramId: 165
+  cosmo:
+    paramId: 500027
+V_10M:
+  ifs:
+    name: 10v
+    paramId: 166
+  cosmo:
+    paramId: 500029
+T_2M:
+  ifs:
+    name: 2t
+    paramId: 167
+  cosmo:
+    paramId: 500011
+TD_2M:
+  ifs:
+    name: 2d
+    paramId: 168
+  cosmo:
+    paramId: 500017
+CLCT:
+  ifs:
+    name: tcc
+    paramId: 164
+  cosmo:
+    paramId: 500046
+W_SNOW:
+  ifs:
+    name: sd
+    paramId: 228141
+  cosmo:
+    paramId: 500044
+
```

Comment:
It is true that FLEXPART-IFS expects SD in m, while FLEXPART-COSMO expects the equivalent parameter in kg m-2 and does the conversion to m in the import. So this is just the inverse of the transformation that we try to implement here. That is what I interpret from readwind_ecmwf.f90 in FLEXPART-IFS and readwind.f90 of FLEXPART-COSMO.
A general conversion from the IFS parameter SD to the COSMO parameter W_SNOW would involve a unit conversion from m to kg m-2. 
The import of FLEXPART must be cleaned up to achieve a clean implementation of the transformations here.